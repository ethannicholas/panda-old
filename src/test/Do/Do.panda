var total := 1
do
    total *= 2
while total < 10000 
Console.writeLine(total)

do {
    total := 0
    break
}
while true
    
Console.writeLine(total)

do
    total += 1
while total < 127 | total % 2 = 1
    
Console.writeLine(total)