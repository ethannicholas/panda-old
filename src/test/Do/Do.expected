16384
0
128
